### DISCUSSION TITLE
How to set MaxConcurrentReconciles for ScaledJobs

### DISCUSSION THREAD

**Discussion by subhamkhemka** [kedacore/keda#3185] [Q&A / Need Help]
Hi

I have 300+ sqs ScaledJob enabled. 
How do I increase MaxConcurrentReconciles?

I am not sure where I need to change the default value for MaxConcurrentReconciles

**Comment by zroubalik** [comment]
https://keda.sh/docs/2.7/operate/cluster/#configure-maxconcurrentreconciles-for-controllers

**Reply by subhamkhemka** [reply]
Thanks, It says - To modify this properties you can set environment variables on both KEDA Operator and Metrics Server Deployments

Can you please help with an example on how to set environment variables for these deployments ?
